SPECS/rubygem-addressable/rubygem-addressable.spec

```diff
@@ -0,0 +1,39 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
+%global gem_name addressable
+Summary:        an alternative implementation to the URI implementation that is part of Ruby's standard library
+Name:           rubygem-addressable
+Version:        2.7.0
+Release:        1%{?dist}
+License:        Apache 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
+Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
```

Comment:
Shouldn't we build the gem ourselves with our own tooling? I'm not sure how gems are built/compiled, but we might be missing some security hardedning our toolchain provides, if we use binaries built externally.

Reply:
All the newly added rubygem packages are implemented the same way as rubygem-bundler, which already exists in Mariner before this PR. I also checked the spec file implementation of rubygems in Fedora and over there, they are also using the official downloaded gem files as the source to build the RPM.